src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write code here
+
+let list = document.querySelectorAll('li');
+
+function sort(ls) {
+  list = [...ls].sort((a, b) => {
```

Comment:
The `sort` function modifies the global `list` variable. It's better to return the sorted list from the function instead of modifying the global variable directly. This will make the function more predictable and easier to test.